### PR TITLE
[ResourceBundle] Add translation to DoctrinePHPCRDriver

### PR DESCRIPTION
**Pull request by andrew-pits**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #4031 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/Doctrine/DoctrinePHPCRDriver.php [hunk 1]
```diff
@@ -49,6 +50,18 @@ protected function addRepository(ContainerBuilder $container, MetadataInterface
             $this->getClassMetadataDefinition($metadata),
         ]);
 
+        if ($metadata->hasParameter('translation')) {
+            $repositoryReflection = new \ReflectionClass($repositoryClass);
+            $translatableRepositoryInterface = TranslatableResourceRepositoryInterface::class;
+            $translationConfig = $metadata->getParameter('translation');
+
+            if (interface_exists($translatableRepositoryInterface) && $repositoryReflection->implementsInterface($translatableRepositoryInterface)) {
```
Comment (stloyd) [inline src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/Doctrine/DoctrinePHPCRDriver.php]: `interface_exists()` should be done in first `if` IMO.

**Comment by stloyd** [comment]
In fact for me current approach for drivers (with translations) is broken, as it should not be in the `ResourceBundle` but in `TranslationBundle`, or I'm missing the point @pjedrzejewski @michalmarcinkowski?

File: src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/Doctrine/DoctrinePHPCRDriver.php (hunk 1 above)
Comment (andrew-pits) [inline src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/Doctrine/DoctrinePHPCRDriver.php]: I think you are right, but i copied this code from DoctrineORMDriver. I though it has to be the same.

**Comment by michalmarcinkowski** [comment]
@stloyd the `TranslationBundle` will be merged into `ResourceBundle` (#3778). The Translations will be still be optional, but the `TranslationBundle` doesn't make sense as standalone and introduces maintenance overhead and 'hacks' like this.

**Comment by stloyd** [comment]
@michalmarcinkowski Ok, kinda missed that PR.

**Comment by pjedrzejewski** [comment]
Awesome, thanks Andrew! I will fix @stloyd's comment in #3778.

**Comment by pjedrzejewski** [comment]
@andrew-pits Congrats on your first PR! :+1:

**Pull request merged**